pr2_description/robots/pr2.urdf.xacro

```diff
@@ -24,7 +24,6 @@
   <xacro:include filename="$(find pr2_description)/urdf/base_v0/base.urdf.xacro" />
   <!-- Head sensors -->
   <xacro:include filename="$(find pr2_description)/urdf/sensors/head_sensor_package.urdf.xacro" />
-  <xacro:include filename="$(find pr2_description)/urdf/sensors/kinect_prosilica_camera.urdf.xacro" />
```

Comment:
Does this affect anything? It was only the prosilica attached to the kinect and not the prosilica camera in the head, yeah?